src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageOverview/StageOverview.component.js

```diff
@@ -91,7 +95,7 @@ export const StageOverviewPlain = ({ title, icon, description, events, classes }
                 content={description}
                 openDelay="100"
             >
-                <div className={classes.indicatorIcon}>
+                <div className={classes.descriptionIcon}>
```

Comment:
This icon is laid out a little differently than the rest, so giving it it's own class.